playbooks/README.md

```diff
@@ -0,0 +1,15 @@
+Automate provisioning OpenStack Instance
+
+- The Ansible playbook will automate the creation of an openstack instance.
+- The playbook contains the neccessary parameters to create the instance. 
+- The parameters include: image, key_name, flavor, network, and security_groups
+- The parameters image, flavor, network, and security groups must be passed in the command line as extra-vars when running the playbook.
+
+- To create an instance, run the following command on the terminal: 
+ansible-playbook provision_openstack_instance.yml -e "vm_name=<give a name for the VM> image=<image name> flavor=<flavor name> network=<network name> sec_groups= <security groups name>"
+
+After running the playbook, you must run the following commands to setup the Open Stack credentials:
+export OS_AUTH_URL=<url-to-openstack-identity> #you can get this url from OS VM
+export OS_PROJECT_NAME=<project-name> 
+export OS_USERNAME=<user-name>
+export OS_PASSWORD=<password>  # (optional)
```

Comment:
For our use case we also require `OS_USER_DOMAIN_NAME` and `OS_PROJECT_DOMAIN_ID` in order for the credential setup to work. That can probably be listed as optional like the `OS_PASSWORD`.